### PR TITLE
scripts[minor]: Add toolkit template

### PR DESCRIPTION
**Pull request by bracesproul**

### CODE REVIEW

File: docs/core_docs/docs/integrations/toolkits/sql.ipynb [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+   "id": "e49f1e0d",
+   "metadata": {},
+   "source": [
+    "# SqlToolkit\n",
+    "\n",
+    "This will help you getting started with the [SqlToolkit](/docs/concepts/#toolkits). For detailed documentation of all SqlToolkit features and configurations head to the [API reference](https://api.js.langchain.com/classes/langchain_agents_toolkits_sql.SqlToolkit.html). You can also find the documentation for the Python equivalent [here](https://python.langchain.com/docs/integrations/toolkits/sql_database/)\n",
+    "\n",
+    "## Setup\n",
+    "\n",
+    "If you want to get automated tracing from runs of individual tools, you can also set your [LangSmith](https://docs.smith.langchain.com/) API key by uncommenting below:\n",
```
Comment (jacoblee93) [inline docs/core_docs/docs/integrations/toolkits/sql.ipynb:28]: Isn't there more setup to do?

File: docs/core_docs/docs/integrations/toolkits/sql.ipynb [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+    "\n",
+    "```typescript\n",
+    "process.env.LANGCHAIN_TRACING_V2=\"true\"\n",
+    "process.env.LANGCHAIN_API_KEY=\"your-api-key\"\n",
+    "```\n",
+    "\n",
+    "### Installation\n",
+    "\n",
+    "This toolkit lives in the `langchain` package. You'll also need to install the `typeorm` peer dependency.\n",
```
Comment (jacoblee93) [inline docs/core_docs/docs/integrations/toolkits/sql.ipynb:37]: These should be moved to community no?

Comment (jacoblee93) [reply docs/core_docs/docs/integrations/toolkits/sql.ipynb:37]: https://python.langchain.com/v0.2/docs/integrations/toolkits/sql_database/

Comment (jacoblee93) [reply docs/core_docs/docs/integrations/toolkits/sql.ipynb:37]: Fine w/ it as a followup

File: docs/core_docs/docs/integrations/toolkits/sql.ipynb [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a38cde65-254d-4219-a441-068766c0d4b5",
+   "metadata": {},
+   "source": [
+    "## Instantiation\n",
+    "\n",
+    "This example uses Chinook database, which is a sample database available for SQL Server, Oracle, MySQL, etc. To set it up, follow [these instructions](https://database.guide/2-sample-databases-sqlite/), placing the `.db` file in the directory where your code lives.\n",
```
Comment (jacoblee93) [inline docs/core_docs/docs/integrations/toolkits/sql.ipynb:58]: These instructions should go under `## Setup`

File: docs/core_docs/docs/integrations/toolkits/sql.ipynb [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e49f1e0d",
+   "metadata": {},
+   "source": [
+    "# SqlToolkit\n",
+    "\n",
+    "This will help you getting started with the [SqlToolkit](/docs/concepts/#toolkits). For detailed documentation of all SqlToolkit features and configurations head to the [API reference](https://api.js.langchain.com/classes/langchain_agents_toolkits_sql.SqlToolkit.html). You can also find the documentation for the Python equivalent [here](https://python.langchain.com/docs/integrations/toolkits/sql_database/)\n",
```
Comment (jacoblee93) [inline docs/core_docs/docs/integrations/toolkits/sql.ipynb:24]: Should add something up here - what does this do? Why is it useful?

File: docs/core_docs/docs/integrations/toolkits/sql.ipynb (hunk 1 above)
Comment (bracesproul) [reply docs/core_docs/docs/integrations/toolkits/sql.ipynb:28]: Ah ya, I had it in the `Instantiation` section but this is prob a better place for it

File: docs/core_docs/docs/integrations/toolkits/sql.ipynb (hunk 2 above)
Comment (bracesproul) [reply docs/core_docs/docs/integrations/toolkits/sql.ipynb:37]: Atm all our toolkits are in langchain, py has them in community

**Pull request merged**